From aaa2eb83b8b6623ddb9cea44445b5342f96100be Mon Sep 17 00:00:00 2001
From: Tulio Magno Quites Machado Filho <tuliom@linux.vnet.ibm.com>
Date: Fri, 24 Nov 2017 18:29:21 -0200
Subject: [PATCH 091/146] powerpc: Update AT_HWCAP2 bits

Linux commit ID cba6ac4869e45cc93ac5497024d1d49576e82666 reserved a new
bit for a scenario where transactional memory is available, but the
suspended state is disabled.

	* sysdeps/powerpc/bits/hwcap.h (PPC_FEATURE2_HTM_NO_SUSPEND): New
	macro.

(cherry picked from commit df0c40ee3a893238ac11f4c0d876a0c3b49d198d)

Signed-off-by: Tulio Magno Quites Machado Filho <tuliom@linux.vnet.ibm.com>
---
 ChangeLog                    | 5 +++++
 sysdeps/powerpc/bits/hwcap.h | 2 ++
 2 files changed, 7 insertions(+)

diff --git a/ChangeLog b/ChangeLog
index 0dcbe3bc69..812c5380f1 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,8 @@
+2017-11-17  Tulio Magno Quites Machado Filho  <tuliom@linux.vnet.ibm.com>
+
+	* sysdeps/powerpc/bits/hwcap.h (PPC_FEATURE2_HTM_NO_SUSPEND): New
+	macro.
+
 2017-08-09  Andreas Schwab  <schwab@suse.de>
 
 	* nptl/Makefile (tests) [$(build-shared) = yes]: Add
diff --git a/sysdeps/powerpc/bits/hwcap.h b/sysdeps/powerpc/bits/hwcap.h
index dfc71c29bb..0668ca041e 100644
--- a/sysdeps/powerpc/bits/hwcap.h
+++ b/sysdeps/powerpc/bits/hwcap.h
@@ -72,3 +72,5 @@
 						 128-bit */
 #define PPC_FEATURE2_DARN	   0x00200000 /* darn instruction.  */
 #define PPC_FEATURE2_SCV	   0x00100000 /* scv syscall.  */
+#define PPC_FEATURE2_HTM_NO_SUSPEND  0x00080000 /* TM without suspended
+						   state.  */
-- 
2.15.1

